packages/replay/src/session/getSession.ts

```diff
@@ -5,59 +5,72 @@ import { createSession } from './createSession';
 import { fetchSession } from './fetchSession';
 import { makeSession } from './Session';
 
-interface GetSessionParams extends SessionOptions {
-  timeouts: Timeouts;
+/**
+ * Check a session, and either return it or a refreshed version of it.
+ * The refreshed version may be unsampled.
+ * You can check if the session has changed by comparing the session IDs.
+ */
+export function maybeRefreshSession(
+  session: Session,
+  {
+    timeouts,
+    traceInternals,
+  }: {
+    timeouts: Timeouts;
+    traceInternals?: boolean;
+  },
```

Comment:
l: This is repeated a few times, worth making it a type?

Reply:
I think we have it twice in here for now, IMHO it's OK to leave it as is as they shouldn't _have_ to be the same (just ended up this way after some refactoring). We can combine it later still :)